src/dsp_tools/commands/excel2json/utils.py

```diff
@@ -78,7 +78,7 @@ def check_contains_required_columns_else_raise_error(
         required_columns: set of column names
 
     Returns:
-        An object if
+        An object if there is a problem else None.
```

Comment:
if you read the prose docstring, the args and returns section get 100% redundant. Could this be removed? Or otherwise the prose section reduced to " This function checks that a dataframe contains all required columns. Additional columns are allowed."?

Reply:
In my opinion, the prose section shouldn't rephrase the args and returns sections.